qiskit/transpiler/passes/layout/dense_layout.py

```diff
@@ -35,39 +35,33 @@ class DenseLayout(AnalysisPass):
         by being set in `property_set`.
     """
 
-    def __init__(self, coupling_map, backend_prop=None):
+    def __init__(self, coupling_map, backend_prop=None, target=None):
```

Comment:
Is there a good way we can make `coupling_map` optional when `target` is provided? I'm guessing the answer is "no", but just curious.